### ISSUE TITLE
Healed bruises and burns never fade

### ISSUE THREAD

**Issue by ArgobargSoup** [VOREStation/VOREStation#582]
#### Brief description of the issue

Basically what it says on the tin.
#### What you expected to happen

Accordin' to the code, or at least the comment in the code, completely healed wounds are supposed ta fade after at least 10 minutes.
#### What actually happened

Still have that tiny, bandaged, bruise I got 1-2 hours ago.
#### Steps to reproduce

-Get bruised/burnt
-Heal the bruise/burn
-Wait for ungodly long time
-Still have bruise/burn
#### Additional info:
- **Server Revision**: release - 2016-09-19
  9da17a2eb407bc85aac6b19a164bfd56fe91818d 
- **Anything else you may wish to add** Judgin' from the code, I think it's a problem with the wound age check calculation.

"organ_external.dm", relevant snippet from "/obj/item/organ/external/proc/update_wounds()":
`for(var/datum/wound/W in wounds)`
`// wounds can disappear after 10 minutes at the earliest`
`if(W.damage <= 0 && W.created + 10 * 10 * 60 <= world.time)`
`wounds -= W`
`continue`
`// let the GC handle the deletion of the wound`

So basically, the later in a round ya get a wound, the longer it would take fer it ta fade.  Probably won't even happen in the round after a certain point.

Was initially gonna suggest that the "heal" mode on bellies would remove minor/healed wounds, then discovered this.

**Comment by SpadesNeil** [comment]
>Server Revision: Latest "VOREstation:master" as of this writing.

This isn't a revision.

**Comment by ArgobargSoup** [comment]
Sorry, figured I had ta put something there.  >_>

I'm workin' with a private test server runnin' the latest version of the code, so I know that this is an issue with the current code...  is the revision still needed in this case?  Test server just returns "unknown".

Current official server revision seems ta be: release - 2016-09-19
9da17a2eb407bc85aac6b19a164bfd56fe91818d

**Comment by Arokha** [comment]
They do now.
